jsonpickle/unpickler.py

```diff
@@ -292,6 +296,28 @@ def loadrepr(reprstr):
     return eval(evalstr, mylocals)
 
 
+def loadmodule(module_str):
+    """Returns a reference to a module.
+
+    >>> obj = loadmodule('datetime/datetime.datetime')
+    >>> obj.__name__
+    'datetime'
+
+    """
+    module, evalstr = module_str.split('/')
```

Comment:
Even though importing arbitrary files might use an eval in it somewhere inside Python, it would probably be best not to name it evalstr if we aren't explicitly eval-ing, to avoid potential confusion from anyone reading the code.